src/main/webapp/app/request/edit-request.html

```diff
@@ -739,10 +739,10 @@ <h4 class="h4">{{ 'Attachments' | translate }}</h4>
 
                     <div class="list-group">
                         <a ng-repeat="file in request.attachments"
-                           ng-href="/requests/{{request.processInstanceId}}/files/{{file.id}}"
+                           ng-href="/api/requests/{{request.processInstanceId}}/files/{{file.id}}"
                            class="list-group-item">
                             <span><i class="glyphicon glyphicon-file"></i></span>
-                            {{file.name}}
+                        /files    {{file.name}}
```

Comment:
why `/files`?

Reply:
Ha, that must be a copy-paste mistake, thanks for spotting